### DISCUSSION TITLE
Sitemap could not be read in search console

### DISCUSSION THREAD

**Discussion by roshaninet** [spatie/laravel-sitemap#385] [Q&A]
hi
why my sitemap could not be read ! i think every thing is ok, but can you help me with this
https://shabesh.com/sitemap.xml

<img width="928" alt="Screen Shot 2021-04-25 at 9 39 03 AM" src="https://user-images.githubusercontent.com/8685071/115981640-6d32c800-a5aa-11eb-9907-854b0b9853d6.png">

**Comment by JJulessNL** [comment]
@roshaninet any solution found for this? Having the same problem.

**Comment by WMspace** [comment]
I'm not ready to hand over my sitemap.xml to GSC yet, but this is a very important issue!

**Comment by michelterstege81** [comment]
Anyone found a solution for this? I'm having the same issue.
https://inpromas.eu/sitemap.xml 

Sitemap cannot be fetched:
![image](https://github.com/spatie/laravel-sitemap/assets/118480982/9ccef0e7-4c59-419d-a643-8ce5b50e4ca2)

Sitemap cannot be read:

![image](https://github.com/spatie/laravel-sitemap/assets/118480982/c20150d0-2ad4-485e-9351-682e1e15dca7)

The url is accessable and returns a 200 statuscode. Checking with an online validator also reveals no errors.

**Comment by JJulessNL** [comment]
I found a solution a while back. You need to specify the sitemap in the robots.txt file. To be sure, you can also add the sitemap to the allow list. So:
```
User-agent: *
Allow: /sitemap.xml
Sitemap: https://<your_domain>/sitemap.xml
```
